llvm/test/tools/repo-fragments/tool-options-test.ll

```diff
@@ -0,0 +1,34 @@
+; Check repo-fragments options, which include:
+;    1) Check the default case by dumping _all_ of the names from a compilation;
+;    2) Check the -names option by dumping one or more fragments by names option;
+;    3) check the -digest-only option;
+;    4) check the -comma option;
+;    5) check the -verbose option;
```

Comment:
The first two “check”s had an initial capital letter. The final semicolon should probably be full-stop to indicate the end of the list.

Reply:
Thanks, fixed.